src/DataSkimmer.jl

```diff
@@ -104,11 +104,13 @@ end
 
 Skim any Tables.jl compatible table.
 """
-function skim(data)::Skimmed
-    if !Tables.istable(data)
+function skim(input_data)::Skimmed
+    if !Tables.istable(input_data)
         throw(ArgumentError("Input to skim() must be a valid Tables.jl table."))
     end
 
+    data = Tables.columns(input_data)
+
     data_schema = Tables.schema(data)
     if isnothing(data_schema)
         column_names = Tables.columnnames(categorical_table)
```

Comment:
I don't see where the `categorical_table` variable is defined?

Reply:
Ah, this is my fault and because this case was never unit tested I didn't catch it. I'll close this for now as it's unrelated to the PR and start a new one that fixes it. Thanks!